src/Compilers/CSharp/Portable/Symbols/SubstitutedNamedTypeSymbol.cs

```diff
@@ -229,7 +229,18 @@ public sealed override ImmutableArray<Symbol> GetMembers()
 
             if (IsTupleType)
             {
-                builder = AddOrWrapTupleMembers(builder.ToImmutableAndFree());
+                var existingMembers = builder.ToImmutableAndFree();
+                var fieldsToRemove = PooledHashSet<Symbol>.GetInstance();
```

Comment:
>PooledHashSet<Symbol>.GetInstance(); [](http://example.com/codeflow?start=37&length=36)

It will probably be more efficient to use identity comparer for this hashset.